packs/data/feat-effects.db/effect-perfect-resistance.json

```diff
@@ -0,0 +1,72 @@
+{
+    "_id": "DyX4E7KDkzRnDxzc",
+    "img": "systems/pf2e/icons/spells/endure-elements.webp",
+    "name": "Effect: Perfect Resistance",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Perfect Resistance]{Perfect Resistance}</p>\n<p>After using a Student of Perfection ki spell, the resistance increases to be equal to your level for 1 round.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 8
+        },
+        "rules": [
+            {
+                "key": "Resistance",
+                "predicate": [
+                    "house-of-perfection:unblinking-flame"
+                ],
+                "type": "fire",
+                "value": "@actor.level"
+            },
```

Comment:
Ok, I have a thought to shrink this down to one RE on each of these effects. If we store the element in actor data using an AE like rule element on the dedication then we can just access it here with `{actor|flags.pf2e.studentOfPerfection.houseElement}` or something.

Reply:
I didn't include "adjustName": false, as it seems like a nice reminder as to what focus spells the character is eligible for.

If you prefer without, I'm happy to add it.